tests/fixture_setup/__init__.py

```
@@ -0,0 +1,45 @@
+import socket
+
+if hasattr(socket, "AF_UNIX"):
+    from ._unix import FakeSnapd  # noqa
```

add the specific code for the ignored error